src/20_Components/amp-geo.html

```diff
@@ -64,25 +64,26 @@
   -->
   <!--
     Here we are using the `amp-geo` grouping feature to define two different groups of ISO country codes: one for those who play soccer and one for those who play football.
-    Notice how we have a group containing 'unknown' in the `amp-geo` tag: if the country cannot be determined, the value is set to 'unknown'. 
+    Notice how we have a group containing 'unknown' in the `amp-geo` tag: if the country cannot be determined, the value is set to 'unknown'.
     When using the grouping feature with `amp-geo` at least one group must contain 'unknown'. When using grouping, `amp-geo` makes available
     a class `amp-geo-group-XX` where XX is the name of the group determined by `amp-geo`.
   -->
   <div>
     The game is called <span class="football"></span>
   </div>
-  
+
   <!--
      ## Integration with amp-bind
   -->
   <!--
-    If the `AMPBind` key is present in the `amp-geo` json configuration, `amp-geo` inserts an `amp-state` tag containing the current ISOCountry and group information.
-  -->
-  <div class="m1">
-    <button class="ampstart-btn caps" on="tap:AMP.setState({myCountry: ampGeo.ISOCountry})">Where am I?</button>
-    <p [text]="'The ISOCountry code is: ' + myCountry "></p>
-    <p [text]="'Matched country groups: ' + ampGeo.ISOCountryGroups.join(', ') "></p>
+    If the `AMPBind` key is present in the `amp-geo` json configuration, `amp-geo` inserts an `amp-state` tag containing the current ISOCountry and group information.  -->
+  <div class="p1">
+    <button class="ampstart-btn caps my1" on="tap:AMP.setState({})">Where am I?</button>
+    <p [text]="'ISOCountry Code: ' + ampGeo.ISOCountry "></p>
+    <p [text]="'Matched Country Groups: ' + ampGeo.ISOCountryGroups.join(', ') "></p>
   </div>
+  <!--  In case you're wondering why the button contains an empty `on="tap:AMP.setState({})"` action, this is to trigger an update of all existing `amp-bind` expressions.
+    `amp-bind` expressions are only evaluated after an user interaction and not on page load. -->
```

Comment:
If you want you could add an explanation why this happens by adding a link to [https://www.ampproject.org/docs/reference/components/amp-bind](https://www.ampproject.org/docs/reference/components/amp-bind) in which it's explained:

> amp-bind does not evaluate expressions on page load, so there's no risk of content jumping unexpectedly. amp-bind also takes special care to ensure speed, security and performance on AMP pages.